routes/web.php

```diff
@@ -68,9 +68,15 @@
 Route::get('/img/{filename}', [HomeController::class, 'getPicture']);
 Route::get('/setlocale/{locale}', [HomeController::class, 'setLocale'])->name('setlocale');
 
-Auth::routes(); //check \Laravel\Ui\AuthRouteMethods
+Auth::routes(['register' => false]); //check \Laravel\Ui\AuthRouteMethods
 
-Route::get('/register/guest', [RegisterController::class, 'showTenantRegistrationForm'])->name('register.guest');
+/* Override the /register route from Auth::routes to force Hungarian */
+Route::prefix('/register')->group(function () {
+    Route::post('/', [RegisterController::class, 'register']);
+
+    Route::get('/', [RegisterController::class, 'showRegistrationForm'])->name('register')->middleware(OnlyHungarian::class);
+    Route::get('/guest', [RegisterController::class, 'showTenantRegistrationForm'])->name('register.guest');
+});
```

Comment:
I would not overcomplicate things, if the Locale Middleware also affects the register page it should be enough. Maybe in the registration page put the description explicitly in hungarian and english as well (so also specify 'hu').

Reply:
I do not understand what you're suggesting. How else would I make `OnlyHungarian` affect `/register`?

Reply:
I was referring to the to app/Http/Middleware/Locale.php